packages/common/container-definitions/src/error.ts

```diff
@@ -84,6 +84,19 @@ export interface ContainerWarning extends IErrorBase {
 
 /**
  * Represents errors raised on container.
+ *
+ * @see
+ *
+ * The following are commonly thrown error types.
```

Comment:
", but the errorType could be any string" or something to confirm that it's not locked down to these (for now)